pkg/cmd/pulumi/convert/convert.go

```diff
@@ -152,7 +153,9 @@ func pclGenerateProject(
 ) (hcl.Diagnostics, error) {
 	_, diagnostics, bindErr := safePclBindDirectory(sourceDirectory, loader, strict)
 	// We always try to copy the source directory to the target directory even if binding failed
-	copyErr := aferoUtil.CopyDir(afero.NewOsFs(), sourceDirectory, targetDirectory)
+	copyErr := aferoUtil.CopyDir(afero.NewOsFs(), sourceDirectory, targetDirectory, func(fi os.FileInfo) bool {
```

Comment:
Last arg can just be nil

Reply:
Yeah changed it, I am always used to passing actual functions xD